dspace/src/main/assembly/release.xml

```diff
@@ -46,6 +46,19 @@
          <!-- Ensure line endings in all files are Unix (LF) line endings-->
          <lineEnding>unix</lineEnding>
       </fileSet>
+      <!-- Ensure the presence of "empty" directories for build steps
+           that need them. -->
```

Comment:
I'd like this comment to be enhanced slightly (as I'm not sure it's descriptive enough of the problem):

<!-- Ensure the presence of "empty" overlay directories for build steps.
       By default our ".gitignore" files are excluded by "useDefaultExcludes=true". -->